src/planet/mpcorb.cpp

```diff
@@ -81,14 +82,23 @@ mpcorb::mpcorb(const std::string& line)
 	tmp.clear();
 	tmp.append(&linecopy[mpcorb_format[8][0]],mpcorb_format[8][1]);
 	boost::algorithm::trim(tmp);
-	m_H = boost::lexical_cast<double>(tmp);
+
+	if (!tmp.empty()) {
+		m_H = boost::lexical_cast<double>(tmp);
+	} else {
+		m_H = std::nan("");
```

Comment:
Change it back to zero as this will not serialized as you found out yesterday ....